README.md

```diff
@@ -25,182 +74,203 @@ repositories {
+                
+                // Suffix which is used to append the line number to the URL. Use #L for GitHub
+                lineSuffix = "#L"
+            }
+    
+            // Used for linking to JDK documentation
+            jdkVersion = 8
+    
+            // Disable linking to online kotlin-stdlib documentation
+            noStdlibLink = false
+            
+            // Disable linking to online JDK documentation
+            noJdkLink = false
+    
+            // Disable linking to online Android documentation (only applicable for Android projects)
+            noAndroidSdkLink = false
+            
+            // Allows linking to documentation of the project's dependencies (generated with Javadoc or Dokka)
+            // Repeat for multiple links
+            externalDocumentationLink {
+                // Root URL of the generated documentation to link with. The trailing slash is required!
+                url = new URL("https://example.com/docs/")
+                
+                // If package-list file is located in non-standard location
+                // packageListUrl = new URL("file:///home/user/localdocs/package-list") 
+            }
+            
+            // Allows to customize documentation generation options on a per-package basis
+            // Repeat for multiple packageOptions
+            perPackageOption {
+                prefix = "kotlin" // will match kotlin and all sub-packages of it
+                // All options are optional, default values are below:
+                skipDeprecated = false
+                reportUndocumented = true // Emit warnings about not documented members 
+                includeNonPublic = false
+            }
+            // Suppress a package
+            perPackageOption {
+                prefix = "kotlin.internal" // will match kotlin.internal and all sub-packages of it
+                suppress = true
+            }
         }
     }
 }
 ```
 
 #### Multiplatform
-Since version 0.10.0 dokka supports multiplatform projects. For a general understanding how a multiplatform documentation is generated, please consult the [FAQ](https://github.com/Kotlin/dokka/wiki/faq).
-In the multiplatform mode, instead of using the `configuration` block, you should use a `multiplatform` block with inner blocks for each platform. 
-The `configuration` block's parameters belong to those inner blocks, which can be named arbitrarly, however if you want to use source roots and classpath provided by Kotlin Multiplatform plugin, 
-they must have the same names as in the Kotlin Multiplatform plugin. See an example below:
+Dokka supports single-platform and multi-platform projects using source sets abstraction. For most mutli-platform projects,
+you should assume that `one platform == one source set`. Source sets can be named arbitrarily, however in order for autoconfiguration
```

Comment:
you should assume that `one platform == one source set`: Isn't this confusing when we talk about something like `jsAndJvmMain` above?

Reply:
Changed